Api/ApiConnection.cs

```diff
@@ -84,7 +85,7 @@ public bool TryRequest<T>(RestRequest request, out T result)
                 // Add the UTC timestamp to the request header.
                 // Timestamps older than 1800 seconds will not work.
                 var timestamp = (int)Time.TimeStamp();
-                var hash = CreateSecureHash(timestamp);
+                var hash = ApiExtensions.CreateSecureHash(timestamp, _token);
```

Comment:
Not an extension; but it works better as a static method on the API -- i.e. Api.CreateSecureHash(timestamp, _token)